Doc/library/dataclasses.rst

```diff
@@ -173,6 +173,11 @@ Module-level decorators, classes, and functions
      glossary entry for details.  Also see the ``dataclasses.KW_ONLY``
      section.
 
+   - ``slots``: If true (the default is ``False``), :attr:`__slots__` attribute
+     will be generated and new class will be returned instead of original one.
```

Comment:
Please change to "instead of the original one" (add "the").

Reply:
Done